applications/autoencoders/data/pilot1/__init__.py

```diff
@@ -0,0 +1,26 @@
+import os
+import os.path
+
+import google.protobuf.text_format
+import lbann
+
+# Paths
+data_dir = os.path.dirname(os.path.realpath(__file__))
+
+# FIXME: Should I add anything to check paths like in cifar10, or
+# download anything like in mnist?
+
```

Comment:
```suggestion
```
I think we can assume the data is where it's supposed to be. Since it's not a common dataset, it'll be on the user to get it right.